portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase

```diff
@@ -132,12 +132,64 @@ definition {
 	}
 
 	@description = "Verify that a link to a URL can be added to the Body Text"
-	@ignore = "true"
 	@priority = "5"
 	test CanSupportALinkToAURL {
 		property portal.acceptance = "false";
 
-		// TODO LPS-131240 CanSupportALinkToAURL pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "1",
+			fieldType = "Paragraph");
+
+		Form.editFieldLabelMetal(fieldValue = "Paragraph");
+
+		Click(
+			toolbox_option = "Link",
+			locator1 = "FormFields#PARAGRAPH_TOOLBOX_EDITOR");
+		
+
+		Click(
+			fieldLabel = "Link Type",
+			optionValue = "url",
+			locator1 = "FormFields#PARAGRAPH_LINK_SELECTOR");
+
+		Click(
+			fieldLabel = "Protocol",
+			optionValue = "http://",
+			locator1 = "FormFields#PARAGRAPH_LINK_SELECTOR");
+
+		Type(
+			value1 = "www.liferay.com",
+			fieldLabel = "URL",
+			locator1 = "FormFields#PARAGRAPH_LINK_TEXT_INPUT");
+
+		Click(locator1 = "CKEditor#OK_BUTTON");
+
+		Click(locator1 = "Button#SAVE");
+
+		Alert.viewSuccessMessage();
+
+		Click(locator1 = "Icon#BACK");
+
+		CommerceNavigator.gotoEntry(entryName = "Form 1");
+
+		Click(
+			key_fieldName = "Paragraph",
+			locator1 = "FormFields#PARAGRAPH_TITLE");
+
+		SelectFrame.selectFrameNoLoading(locator1 = "CKEditor#BODY_FIELD_IFRAME");
+
+		AssertElementPresent(
+			url = "http://www.liferay.com",
+			locator1 = "FormFields#PARAGRAPH_LINK_TEXT_BODY");
```

Comment:
@Jamersonze, I think that all these 3 steps can be handled at once by `FormFields.viewParagraphField`, can you please confirm?

Maybe, this macro just need a little change to attend your need. :wink:

Reply:
Fixed